include/swift/Reflection/TypeRefBuilder.h

```diff
@@ -982,35 +991,46 @@ class TypeRefBuilder {
   IntVariableReader OpaqueIntVariableReader;
 
 public:
-  template<typename Runtime>
```

Comment:
Looks like this is just a formatting change. Can we do that as a separate PR if so?

Reply:
It got formatted because of the new `remote::ExternalTypeRefInfoProvider *provider = nullptr` parameter, but I guess it formatted the whole constructor 😅  I'll remove the non-related parts.